backend/src/gpml/handler/stakeholder.clj

```diff
@@ -359,44 +357,61 @@
             response
             (assoc-in response [:body :error-details :error] (.getMessage e))))))))
 
-(def ^:const suggested-profiles-per-page 5)
+(def ^:const default-suggested-profiles-per-page 5)
+(def ^:const default-get-suggested-profiles-page 0)
 
-(defn api-suggested-profiles-opts->suggested-profiles-opts
-  [suggested-profiles-opts]
-  (util/update-if-not-nil suggested-profiles-opts :page #(Integer/parseInt %)))
+(defn- api-suggested-profiles-opts->suggested-profiles-opts
```

Comment:
I feel like we should start specifying default values in the Malli schema for the parameters instead of having a specific function to do that. What do you think @joseAyudarte91 ?

Reply:
Yep, I actually was thinking that there were some issues specifying the default values in the params spec, but if we can do it without issues makes total sense :) I remember you said something about optional and default combinations being problematic but cannot remember. We can discuss in a call.

Reply:
We can do it without issues. The problem is I understood the default values approach wrong. When you say that some property has a default value you can't say it's optional too. If you do make it optional it will ignore the default value. In short this would work:

```clj
[:map 
 [:page {:default 0} pos-int?]]
```

But this wouldn't:
```clj
[:map 
 [:page {:default 0 :optional true} pos-int?]]
```

Reply:
Ah ok, now I remember yes, thanks. Just tried and it works fine, I will update the PR.

Reply:
Ready, I will merge it as we have agreed.